submarine-workbench/interpreter/interpreter-engine/pom.xml

```diff
@@ -27,7 +27,7 @@
     <groupId>org.apache.submarine</groupId>
     <artifactId>interpreter</artifactId>
     <version>0.3.0-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
+    <relativePath>..</relativePath>
```

Comment:
Why do we need to change "../pom.xml" to ".."?
../pom.xml is default value of relativePath